frontend/express/public/core/events/javascripts/countly.common.components.js

```diff
@@ -0,0 +1,42 @@
+/*global window, countlyVue, Vue */
+(function (countlyCommonEvents) {
+    countlyCommonEvents.helpers = {
+    };
+    var _mixins = countlyVue.mixins;
+    Vue.component("cly-events-breakdown-horizontal-tile", countlyVue.views.BaseView.extend({
+        mixins: [
+            _mixins.i18n
```

Comment:
i dont think you are using the i18n mixin in the template, so maybe we can remove it.